package.json

```diff
@@ -1,11 +1,6 @@
 {
   "name": "bonfhir",
   "private": true,
-  "workspaces": [
-    "apps/*",
-    "packages/*",
-    "docs/*"
-  ],
```

Comment:
Dont commit this. 

I was getting an error when trying to run `npm install` related to this. Have not yet investigate the solution